packages/framer-motion/src/value/index.ts

```diff
@@ -419,6 +424,32 @@ export class MotionValue<V = any> {
         this.clearAnimation()
     }
 
+    /**
```

Comment:
Rather than increase the API surface of `MotionValue` it would be preferable to just use the active animation directly.